data-access-analysis-tools/index-rsp-warnings.rst

```diff
@@ -12,98 +12,60 @@
 
-Data Sharing and Safeguards
----------------------------
-
-A number of safeguards for avoiding uptime or temporary data loss will not be present -- the resources are still in “trusted user” mode.
-
-The fact that anyone could PRUNE, there's no provenance for destruction, and how to avoid accidental wipe-outs.
-
-Do we have shared data sets? We did on the NCSA RSP. (HFC: what kinds of shared datasets are you thinking here? The only data set we officially provide is the DP0.1 data set. There are a lot more data on NCSA's filesystem and we never intended to have them on IDF.)
-
-Do we have access to other users’ home directories?
-
-Need some warnings about per-user data quota. 
-
-Need some warnings about shared-write area.
-
-
-
-RSP Compute Performance
------------------------
-
-Performance during DP0 may not reflect the performance of the final system (see also Q4), and the resources made available to DP0 delegates may not reflect the final user quotas of the operations-era RSP.
-
-How about running cron jobs? (HFC: I don't think we would let delegates run cron. But what is the motivation of wanting to run cron?)
-
-What about abusing resources?
+User data: No parachutes!
+-------------------------
 
-Testing how algorithms will scale: No access to user batch or parallelisation are available in DP0. Resources provided as part of DP0.1 are limited and meaningful scalability testing of algorithms is not practical. 
+When you access the RSP Notebook aspect, you have a home space you can use to store your notebooks, scripts, extra python modules and small files. Other users cannot see the contents of your home space, but you have no expectation of privacy from observatory staff monitoring the systems. If we experience an incident, we will attempt to restore the contents of your homespace. The contents of your homespace will not be deleted without attempting to notify you.
 
+Data in /scratch is not backed up.
 
+Data in /projects is available for use in the situation where you wish to share files with other RSP users. For this data preview, any files in you put in /projects are readable and writeable by all users. Take care you do not delete other people's files.
 
-Butler-Related Warnings
------------------------
+For this data preview there are no file quotas applied, however usage will be monitored and users may be asked to reduce their footprint, either in their home space or shared data spaces. By using the Butler to write data, you are not using disk space but our data facility's object stores; ample capacity is available there and you should use that in favour of keeping large data files on disk.
 
-For DP0.1 or until we have a client/server Butler and signed URLs (DMTN-169,DMTN-176), all users share full read/write without restriction.  Since there is no individual access control, absolutely no use direct sql access will be allowed to any butler registry database.
```

Comment:
Why are you deleting my admonishment about using direct sql access to registry? I asked for this original so people couldn't come back and tell us they didn't know they weren't supposed to login to registry and run a delete to fix something up.

Reply:
Some kind of warning like this seems necessary.

Reply:
It feels like "let me explain the thing we don't want you to do, and also plant the idea in your head that you didn't have". We specifically tell people in two places to only interact with data through the Butler APIs. I myself would not have known I could use direct sql access. Do you really want to spell it out?

Reply:
PS. I don't want to cut off debate on this but given the state of the original content and the timeline I am going to go ahead and merge this PR in. Tim - by all means you are welcome to PR any further changes you like.